pkg/odo/cli/service/create.go

```diff
@@ -45,46 +49,96 @@ func NewServiceCreateOptions() *ServiceCreateOptions {
 
 // Complete completes ServiceCreateOptions after they've been created
 func (o *ServiceCreateOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
-	o.serviceType = args[0]
-	// if only one arg is given, then it is considered as service name and service type both
-	o.serviceName = o.serviceType
-	// if two args are given, first is service type and second one is service name
-	if len(args) == 2 {
-		o.serviceName = args[1]
+	if len(args) == 0 || !cmd.HasFlags() {
+		o.interactive = true
 	}
+
 	o.Context = genericclioptions.NewContextCreatingAppIfNeeded(cmd)
-	return err
+
+	client := o.Client
+
+	var class scv1beta1.ClusterServiceClass
+	if o.interactive {
+		classesByCategory, _ := client.GetServiceClassesByCategory()
```

Comment:
Perhaps we could add a spinner here? 
I am mentioning it because it took a few seconds to populate this on some occations

Reply:
OK